JiraReporter/Services/AuthorLoader.cs

```diff
@@ -391,19 +396,39 @@ private Image GetImageFromURL(string url)
                 EncryptionService = EncryptionService
             }.Authorize(webClient,_context.JiraRequestContext, UrlExtensions.GetRelativeUrl(url));
 
-            var imageData = webClient.DownloadData(url);
+            return webClient.DownloadData(url);
+        }
+
+        private void SetUserAvatar(byte[] image)
+        {
+            using(var db = new ReportsDb())
```

Comment:
La fel si aici. Tu nu lurezi direct cu ReportsDb in JiraReporter. Tu lucrezi doar prin intermediul serviciilor. 

Nici nu ar trebui sa ai referinta catre DAL in aceasta aplicatie.